Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ListWSController.java

```diff
@@ -335,6 +335,7 @@ public String getWorkspaceDetails(@PathVariable("workspaceid") String workspacei
 		
 		//Including a condition to check if the workspace is not deactive. If the workspace is deactive adding attribute to make delete button disabled
 		model.addAttribute("isDeactivated", wsManager.getDeactiveStatus(workspaceid));
+        model.addAttribute("isArchived", wsManager.getArchiveState(workspaceid));

```

Comment:
Fixed the formatting. My IDE uses 4 spaces as the indentation. I changes the settings